features/app/UI/Gallery/PopUpImage/PopUpImage.module.scss

```
@@ -0,0 +1,137 @@
+@import 'shared/styles/breakpoints';
+
+.root {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--white);
+  display: flex;
+  align-items: center;
+  z-index: 999;
+}
+
+.cross {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 60px;
+  height: 60px;
+  top: 10px;
+  right: 10px;
+  background: transparent;
+  z-index: 999;
+  cursor: pointer;
+
+  @media (any-hover: hover) {
+    &:hover {
+      &:after {
+        background-color: var(--gray);
+      }
+      &:before {
```

```suggestion
      &:before,
      &:after {
```
сверху удалить кусочек

---

Готово